src/server/DRL_parameters.py

```diff
@@ -0,0 +1,25 @@
+class DRL_parameters:
+    def __init__(self,
+        nb_epochs: int,
+        learning_rate: float,
+        batch_size: int,
+        momentum: float) :
+        self.__nbEpochs = nb_epochs
```

Comment:
In python, the norm is to name attributes using the sanke_case. For example `__nbEpocks` gives `__nb_epochs`

Reply:
You're right. I just wanted to code the parameters and attributes differently, so I will reverse it.